### DISCUSSION TITLE
和 SoLiD 项目的相似性和不同之处

### DISCUSSION THREAD

**Discussion by linonetwo** [buckyos/CYFS#35] [Q&A] [answered]
初步了解 CYFS 应该也是个 BaaS ，而且相信你们也是语义网爱好者，那你们肯定也了解过 TBL 爵士的 SoLiD 项目，想了解一下你们理解的与它的异同，以及在吸引商业和开源开发者两方面与它的竞争计划，谢谢。

**Comment by alexsunxl** [comment]
你好，感谢提问。  
这个问题更欢迎你到我们的discord频道来了解和交流  https://discord.com/invite/cyfs。   
issue里面可能倾向于代码问题

**Reply by linonetwo** [reply]
你好，这不是 issue，是discussion，很适合提容易被google到的问题

**Reply by linonetwo** [reply]
是不方便公开吗？

**Reply by alexsunxl** [reply]
discord 也是公开的。 是说discord 频道上有更多人给你回复。这边可能更专注开发相关的问题。

**Reply by linonetwo** [reply]
算是开发问题，我在做技术选型，因为没搞清楚这个，目前先用 solid 了

**Comment by waterflier** [comment, marked as answer]
CYFS 和BaaS一点关系都没有，SoLiD是CYFS的重要原型之一，我们在设计CYFS的时候对SoLiD进行了仔细的研究，包括最重要的术语OOD也是和POD同源的。

最核心的区别是两点
1. 我们不依赖DNS系统，使用DNS协议必然导致中心化
2. 我们的语义网强调确权，数据的确权是实现价值网络的基础。这部分的设计来自BTC，SoLiD并没有这方面的设计。

**Reply by linonetwo** [reply]
感谢解答，那我了解了，可能 CYFS 是比 SoLiD 更好的选型，我让我队友调研一下。

**Reply by linonetwo** [reply]
当然如果都采用了 LDP OIDC 等协议，应该和 SoLiD 账号是可以互相打通的吧？我们目前在用 solid，之后可以看看对接替换成本高不高。
